sauce/features/general/hide-help/index.js

```diff
@@ -0,0 +1,60 @@
+import { Feature } from 'core/feature';
+
+export class hideHelp extends Feature {
+
+  injectCSS() { return require('./index.css'); }
+
+  shouldInvoke() {
+    return true;
+  }
```

Comment:
What if the user hasn't enabled this feature? The code that calls shouldInvoke() doesn't check that the feature is enabled so I believe it should be check here.
```
    featureInstances.forEach(feature => {
      if (feature.settings.enabled && feature.injectCSS()) {
        globalCSS += `/* == Injected CSS from feature: ${feature.constructor.name} == */\n\n${feature.injectCSS()}\n`;
      }
    });

    ynabToolKit.invokeFeature = (featureName) => {
      featureInstances.forEach((feature) => {
        if (feature.constructor.name === featureName && feature.shouldInvoke()) {
          feature.invoke();
        }
      });
    };
```